### PR TITLE
Improve rate limit policy

### PR DESCRIPTION
**Pull request by y-tabata**
This PR improves the rate limit policy according to the comments of PR #665.
From #703.

### CODE REVIEW

File: gateway/src/apicast/policy/rate_limit/apicast-policy.json [hunk 1]
```diff
@@ -23,18 +23,14 @@
                 "scope": {
                   "type": "string",
                   "description": "Scope of the key",
-                  "default": "global",
+                  "default": "service",
```
Comment (davidor) [inline gateway/src/apicast/policy/rate_limit/apicast-policy.json]: I think this change of behavior should be documented in the changelog.

Comment (davidor) [reply gateway/src/apicast/policy/rate_limit/apicast-policy.json]: This policy was included in a 'release candidate' version that can become final. If it does, this change will be included in the next release and be considered a breaking change. I think we should be clear about that in the changelog.

**Comment by davidor** [comment]
Apart from what I commented above, the PR looks good to me :+1:

File: gateway/src/apicast/policy/rate_limit/apicast-policy.json (hunk 1 above)
Comment (mikz) [reply gateway/src/apicast/policy/rate_limit/apicast-policy.json]: @davidor I'm not sure if it is a breaking change. Just changing the default value will affect only new instances of this policy.

Comment (davidor) [reply gateway/src/apicast/policy/rate_limit/apicast-policy.json]: You're right @mikz :+1: 
This is not a breaking change, sorry for the confusion.

**Pull request merged**